pkg/virt-launcher/virtwrap/nichotplug.go

```diff
@@ -61,13 +76,12 @@ func hotplugVirtioInterface(vmi *v1.VirtualMachineInstance, converterContext *co
 			return fmt.Errorf("could not retrieve the api.Interface object from the dummy domain")
 		}
 
-		log.Log.Infof("will hot plug %s with MAC %s", network.Name, *relevantIface.MAC)
```

Comment:
Because the unit tests provide status interfaces without MAC addresses, which would fail the tests (nil pointer dereference).

I have the alternative of either providing a MAC on the unit tests, adding production code to not print the log when the mac is not available, or, simply removing the log. I chose the latter option.

Would you choose another one ? A close second for me would be to provide the MAC from the unit tests.

Thoughts @AlonaKaplan ?